kinfraglib/filters/utils.py

```diff
@@ -48,6 +48,7 @@ def add_values(fragment_library, values, colname):
     pocket_num = (
         0  # helper variable to count which subpocket index is the current index
     )
+    values = [val for val in values if len(val)] # colums that are not empty
     for subpocket in fragment_library.keys():
```

Comment:
Just to clarify so that I understand the bug - in the case of an empty subpocket after filtering, `fragment_library` already contains the filtered library, right? This means that `fragment_library.keys()` will no longer contain the empty subpocket, which is why we have to remove the empty lists in `values` - correct? Using `pocket_num` as an index for `values` and matching that to the subpockets seems quite error-prone 😄